bip-0197.mediawiki

```diff
@@ -0,0 +1,156 @@
+<pre>
+  BIP: 197
+  Layer: Applications
+  Title: Hashed Time-Locked Collateral Contract
+  Author: Matthew Black <matthew@atomicloans.io>
+          Tony Cai <tony@atomicloans.io>
+          Atomic Loans team <info@atomicloans.io>
```

Comment:
Not sure teams can be authors; either way, this needs to match what's in README

Reply:
Removed `Atomic Loans team <info@atomicloans.io>` from Author list